### PR TITLE
Add RGB, ARGB and RGBA methods for the category of UIColor.

### PR DESCRIPTION
**Pull request by joehsieh**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

File: AppDevPods/AppDevCommonKit/UIColor+ADKHexPresentation.h [hunk 1]
```diff
@@ -37,7 +37,8 @@
 - (UIColor *)ADKInitWithHexRed:(NSUInteger)red green:(NSUInteger)green blue:(NSUInteger)blue alpha:(CGFloat)alpha;
 
 /**
- * @brief Create a color by hex string.
+ * @brief Create a color by hex string. The returned color is undefined if hexString is invalid.
+ *        The behaviour is the same as ADKColorWithRGBHexString:, but ADKColorWithRGBHexString is preferred.
```
Comment (qcl) [inline AppDevPods/AppDevCommonKit/UIColor+ADKHexPresentation.h]: Second `ADKColorWithRGBHexString` missed its `:`.

**Review by qcl — CHANGES_REQUESTED** [review]
I'd done my review. And I hope you can also complete the test for those new methods.

**Comment by joehsieh** [comment]
I've fixed the typo and applied related tests.

**Comment by joehsieh** [comment]
BTW, I also upgrade IPHONEOS_DEPLOYMENT_TARGET of test target from 7.0 to 8.0 to build the tests successfully.

File: AppDevKitTests/AppDevCommonKit/UIColor+ADKHexPresentationSpec.m [hunk 2]
```diff
@@ -77,4 +77,25 @@
     });
 });
 
-SPEC_END
\ No newline at end of file
+describe(@"Test ADKColorWithRGBHexString", ^{
+    it(@"given color with 7b0099, should get string 7B0099", ^{
```
Comment (qcl) [inline AppDevKitTests/AppDevCommonKit/UIColor+ADKHexPresentationSpec.m]: It should be `given string 7B0099, should get color with 7b0099`.

File: AppDevKitTests/AppDevCommonKit/UIColor+ADKHexPresentationSpec.m [hunk 3]
```diff
@@ -77,4 +77,25 @@
...
+        UIColor *expectedColor = [UIColor colorWithRed:123.0f / 255.0f green:0.0 / 255.0f blue:153.0f / 255.0f alpha:1.0f];
+        [[[UIColor ADKColorWithRGBHexString:@"7B0099"] should] equal:expectedColor];
+    });
+});
+
+describe(@"Test ADKColorWithARGBHexString", ^{
+    it(@"given color with EE7b0099, should get string EE7B0099", ^{
```
Comment (qcl) [inline AppDevKitTests/AppDevCommonKit/UIColor+ADKHexPresentationSpec.m]: Ditto.

File: AppDevKitTests/AppDevCommonKit/UIColor+ADKHexPresentationSpec.m [hunk 4]
```diff
@@ -77,4 +77,25 @@
...
+        UIColor *expectedColor = [UIColor colorWithRed:123.0f / 255.0f green:0.0f / 255.0f blue:153.0f / 255.0f alpha:238.0f / 255.0f];
+        [[[UIColor ADKColorWithARGBHexString:@"EE7B0099"] should] equal:expectedColor];
+    });
+});
+
+describe(@"Test ADKColorWithRGBAHexString", ^{
+    it(@"given color with 7b0099EE, should get string 7B0099EE", ^{
```
Comment (qcl) [inline AppDevKitTests/AppDevCommonKit/UIColor+ADKHexPresentationSpec.m]: Ditto.

**Review by qcl — CHANGES_REQUESTED** [review]
I'd done my review.

**Comment by joehsieh** [comment]
I've updated the test description. Thanks.

**Review by qcl — APPROVED** [review]
👍

**Pull request merged**